conformance/reports/v1.2.1/traefik-traefik/experimental-v3.2.2-default-report.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: gateway.networking.k8s.io/v1
+date: '-'
```

Comment:
Why no date here? Is there any specific reason to omit it?

Reply:
As the conformance report is versioned, we removed the date to avoid errors in our CI when we check that the report is unchanged/up-to-date. We have this in mind and want to fix it.

Reply:
Sounds good. I think it's not a big deal the lack of a date here.